bedrock/firefox/templates/firefox/challenge-the-default/landing.html

```diff
@@ -46,18 +46,10 @@
           'loading': 'eager',
           'height': '24',
           'class': 'ctd-sub-nav-image',
-          'alt': 'Firefox'
+          'alt': 'Challenge the Default'
```

Comment:
I was meaning it should read “Firefox - Challenge the Default”, to match the text that is visually displayed.

Reply:
ah! got it